app/src/main/java/org/wikipedia/commons/FilePage.kt

```diff
@@ -0,0 +1,13 @@
+package org.wikipedia.commons
+
+import org.wikipedia.dataclient.mwapi.MwQueryPage
+
+class FilePage {
+    var thumbnailWidth = 0
+    var thumbnailHeight = 0
+    var imageFromCommons = false
+    var showEditButton = false
+    var showFilename = false
+    var page = MwQueryPage()
+    var imageTags = emptyMap<String, List<String>>()
```

Comment:
Can all of the above fields be constructor parameters?
And in the places where you create instances of `FilePage`, use the constructor instead of `.apply { }`?

Reply:
Done!